app/assets/stylesheets/global.css.scss

```diff
@@ -60,18 +60,17 @@ button.button a {
 }
 
 .actions input {
-  background-color: #1583cc;
+  background-color: #022902;
```

Comment:
Color literals like `#022902` should only be used in variable declarations; they should be referred to via variable everywhere else.